class.jetpack-heartbeat.php

```diff
@@ -162,27 +122,63 @@ public static function generate_stats_array( $prefix = '' ) {
 		return $return;
 	}
 
+	/**
+	 * Registers jetpack.getHeartbeatData xmlrpc method
+	 *
+	 * @deprecated since 8.9.0
+	 * @see Automattic\Jetpack\Heartbeat::jetpack_xmlrpc_methods()
+	 *
+	 * @param array $methods The list of methods to be filtered.
+	 * @return array $methods
+	 */
 	public static function jetpack_xmlrpc_methods( $methods ) {
-		$methods['jetpack.getHeartbeatData'] = array( __CLASS__, 'xmlrpc_data_response' );
-		return $methods;
+		return Heartbeat::jetpack_xmlrpc_methods( $methods );
```

Comment:
Should we add a deprecation notice here and below, before we return the new method?

Reply:
Yes, I think in those 2 cases we are fine. They are not used anywhere else